From b04f0afb74b8ab699113b74b62e14bf9b74737ff Mon Sep 17 00:00:00 2001
From: Michael Achenbach <machenbach@chromium.org>
Date: Thu, 25 May 2023 18:17:26 +0200
Subject: [PATCH] [test] Migrate linux testing from Bionic to Jammy

Cq-Include-Trybots: luci.v8.try:v8_numfuzz_dbg
Cq-Include-Trybots: luci.v8.try:v8_numfuzz_rel
Cq-Include-Trybots: luci.v8.try:v8_numfuzz_tsan_rel
Cq-Include-Trybots: luci.v8.try:v8_linux_noi18n_rel
Cq-Include-Trybots: luci.v8.try:v8_linux64_msan_rel
Cq-Include-Trybots: luci.v8.try:v8_linux64_ubsan_rel
Bug: chromium:1448884
Change-Id: Ib3de10c84f82d838458b8dde8deab913161b4d59
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4568005
Reviewed-by: Liviu Rau <liviurau@google.com>
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Cr-Commit-Position: refs/heads/main@{#87874}
---
 infra/testing/builders.pyl | 230 ++++++++++++++++++-------------------
 1 file changed, 115 insertions(+), 115 deletions(-)

diff --git a/infra/testing/builders.pyl b/infra/testing/builders.pyl
index 42c79518de..1d511b5f21 100644
--- a/infra/testing/builders.pyl
+++ b/infra/testing/builders.pyl
@@ -48,7 +48,7 @@
   # Fuchsia
   'v8_fuchsia_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'fuchsia-unittests'},
@@ -56,7 +56,7 @@
   },
   'V8 Fuchsia': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'fuchsia-unittests'},
@@ -67,7 +67,7 @@
   'v8_linux_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -112,7 +112,7 @@
   },
   'v8_linux_gc_stress_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit', 'variant': 'slow_path', 'test_args': ['--gc-stress'], 'shards': 2},
@@ -122,7 +122,7 @@
   'v8_linux_nodcheck_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -146,7 +146,7 @@
   },
   'v8_linux_noi18n_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mozilla', 'variant': 'default'},
@@ -157,7 +157,7 @@
   'v8_linux_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -186,7 +186,7 @@
   'v8_linux_optional_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       # Code serializer.
@@ -292,7 +292,7 @@
   },
   'v8_linux_verify_csa_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -302,7 +302,7 @@
   # Linux32 with arm simulators
   'v8_linux_arm_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access', 'shards': 3},
@@ -314,7 +314,7 @@
   },
   'v8_linux_arm_lite_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default', 'shards': 4},
@@ -322,7 +322,7 @@
   },
   'v8_linux_arm_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access', 'shards': 2},
@@ -357,7 +357,7 @@
   },
   'v8_linux64_asan_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'shards': 12},
@@ -368,7 +368,7 @@
   },
   'v8_linux64_cfi_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -380,7 +380,7 @@
   },
   'v8_linux64_coverage_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default'},
@@ -389,7 +389,7 @@
   },
   'v8_linux64_coverage_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default'},
@@ -399,7 +399,7 @@
   'v8_linux64_cppgc_non_default_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -407,7 +407,7 @@
   },
   'v8_linux64_css_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 5},
@@ -416,7 +416,7 @@
   'v8_linux64_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks', 'shards': 2},
@@ -450,7 +450,7 @@
   'v8_linux64_dict_tracking_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -458,7 +458,7 @@
   },
   'v8_linux64_disable_runtime_call_stats_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -467,7 +467,7 @@
   'v8_linux64_external_code_space_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -475,14 +475,14 @@
   },
   'v8_linux64_fuzzilli_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     # TODO(almuthanna): Add a new test config for the fuzzilli suite.
     'tests': [],
   },
   'v8_linux64_fyi_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       # Infra staging.
@@ -501,7 +501,7 @@
   },
   'v8_linux64_gc_stress_custom_snapshot_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -514,7 +514,7 @@
   'v8_linux64_gc_stress_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'd8testing', 'test_args': ['--gc-stress'], 'shards': 5},
@@ -549,7 +549,7 @@
   'v8_linux64_minor_mc_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'minor_mc'},
@@ -561,7 +561,7 @@
   },
   'v8_linux64_msan_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'variant': 'default', 'shards': 4},
@@ -571,7 +571,7 @@
   'v8_linux64_nodcheck_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -600,7 +600,7 @@
   },
   'v8_linux64_perfetto_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -608,7 +608,7 @@
   },
   'v8_linux64_no_pointer_compression_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -617,7 +617,7 @@
   'v8_linux64_no_sandbox_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 5},
@@ -626,7 +626,7 @@
   'v8_linux64_official_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -634,7 +634,7 @@
   },
   'v8_linux64_single_generation_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -643,7 +643,7 @@
   'v8_linux64_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       # TODO(machenbach): Add benchmarks.
@@ -683,7 +683,7 @@
   'v8_linux64_predictable_rel': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -694,7 +694,7 @@
   'v8_linux64_no_sandbox_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -702,7 +702,7 @@
   },
   'v8_linux64_tsan_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks', 'shards': 2},
@@ -716,7 +716,7 @@
   },
   'v8_linux64_tsan_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks', 'shards': 2},
@@ -728,7 +728,7 @@
   },
   'v8_linux64_tsan_no_cm_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -746,7 +746,7 @@
   },
   'v8_linux64_tsan_isolates_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'test_args': ['--isolates'], 'shards': 7},
@@ -754,7 +754,7 @@
   },
   'v8_linux64_ubsan_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -762,7 +762,7 @@
   },
   'v8_linux64_verify_csa_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -772,7 +772,7 @@
   # Linux64 with arm64 simulators
   'v8_linux_arm64_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access', 'shards': 2},
@@ -784,7 +784,7 @@
   },
   'v8_linux_arm64_gc_stress_dbg': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'd8testing', 'test_args': ['--gc-stress'], 'shards': 12},
@@ -792,7 +792,7 @@
   },
   'v8_linux_arm64_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access', 'shards': 4},
@@ -805,7 +805,7 @@
   },
   'v8_linux_arm64_cfi_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'variant': 'default', 'shards': 3},
@@ -814,7 +814,7 @@
   },
   'v8_linux64_arm64_no_pointer_compression_rel': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default', 'shards': 4},
@@ -824,7 +824,7 @@
   # Linux64 with Loongson simulators
   'v8_linux64_loong64_rel': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -834,7 +834,7 @@
   # Linux with RISC-V simulators
   'v8_linux_riscv32_rel': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -842,7 +842,7 @@
   },
   'v8_linux64_riscv64_rel': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -1085,7 +1085,7 @@
   # Main.
   'V8 Fuzzer': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -1099,7 +1099,7 @@
   'V8 Linux': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1171,7 +1171,7 @@
   },
   'V8 Linux - arm64 - sim - CFI': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'variant': 'default', 'shards': 3},
@@ -1180,7 +1180,7 @@
   },
   'V8 Linux - arm64 - sim - MSAN': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'variant': 'default', 'shards': 4},
@@ -1190,7 +1190,7 @@
   'V8 Linux - debug': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1293,7 +1293,7 @@
   },
   'V8 Linux - full debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default', 'shards': 6},
@@ -1301,7 +1301,7 @@
   },
   'V8 Linux - gc stress': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -1319,7 +1319,7 @@
   },
   'V8 Linux - noi18n - debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mozilla', 'variant': 'default'},
@@ -1330,7 +1330,7 @@
   'V8 Linux64 - predictable': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1340,7 +1340,7 @@
   },
   'V8 Linux - shared': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mozilla'},
@@ -1350,7 +1350,7 @@
   },
   'V8 Linux - verify csa': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -1358,7 +1358,7 @@
   },
   'V8 Linux PGO instrumentation - builder' : {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'pgo_instrumentation'}
@@ -1367,7 +1367,7 @@
   'V8 Linux64': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1432,7 +1432,7 @@
   'V8 Linux64 - official': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -1440,7 +1440,7 @@
   },
   'V8 Linux64 - cfi': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1452,7 +1452,7 @@
   },
   'V8 Linux64 - coverage': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default'},
@@ -1461,7 +1461,7 @@
   },
   'V8 Linux64 - coverage - debug': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default'},
@@ -1470,7 +1470,7 @@
   },
   'V8 Linux64 - custom snapshot - debug': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit', 'test_args': ['--no-harness']},
@@ -1479,7 +1479,7 @@
   'V8 Linux64 - debug': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks'},
@@ -1527,7 +1527,7 @@
   'V8 Linux64 - minor mc - debug': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'minor_mc'},
@@ -1539,7 +1539,7 @@
   },
   'V8 Linux64 - disable runtime call stats': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -1547,7 +1547,7 @@
   },
   'V8 Linux64 - debug - fyi': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       # Infra staging.
@@ -1567,7 +1567,7 @@
   'V8 Linux64 - cppgc-non-default - debug': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -1575,7 +1575,7 @@
   },
   'V8 Linux64 - debug - perfetto': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -1588,7 +1588,7 @@
   },
   'V8 Linux64 - debug - single generation': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -1602,7 +1602,7 @@
   'V8 Linux64 - external code space - debug': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 3},
@@ -1610,7 +1610,7 @@
   },
   'V8 Linux64 - fyi': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       # Infra staging.
@@ -1638,7 +1638,7 @@
   'V8 Linux64 - gc stress': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -1668,7 +1668,7 @@
   },
   'V8 Linux64 - internal snapshot': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -1676,7 +1676,7 @@
   },
   'V8 Linux64 - no pointer compression': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -1685,7 +1685,7 @@
   'V8 Linux64 - no sandbox': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -1694,7 +1694,7 @@
   'V8 Linux64 - no sandbox - debug': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 2},
@@ -1702,7 +1702,7 @@
   },
   'V8 Linux64 - shared': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mozilla'},
@@ -1712,7 +1712,7 @@
   },
   'V8 Linux64 - verify csa': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing'},
@@ -1720,7 +1720,7 @@
   },
   'V8 Linux64 ASAN': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'test262', 'shards': 7},
@@ -1731,7 +1731,7 @@
   },
   'V8 Linux64 css - debug': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'shards': 5},
@@ -1739,7 +1739,7 @@
   },
   'V8 Linux64 GC Stress - custom snapshot': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -1751,7 +1751,7 @@
   },
   'V8 Linux64 PGO instrumentation - builder' : {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'pgo_instrumentation'}
@@ -1759,7 +1759,7 @@
   },
   'V8 Linux64 TSAN': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks', 'shards': 2},
@@ -1773,7 +1773,7 @@
   },
   'V8 Linux64 TSAN - debug': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'benchmarks', 'shards': 2},
@@ -1785,7 +1785,7 @@
   },
   'V8 Linux64 TSAN - stress-incremental-marking': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -1818,7 +1818,7 @@
   },
   'V8 Linux64 TSAN - isolates': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'test_args': ['--isolates'], 'shards': 7},
@@ -1826,7 +1826,7 @@
   },
   'V8 Linux64 TSAN - no-concurrent-marking': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -1844,7 +1844,7 @@
   },
   'V8 Linux64 UBSan': {
     'swarming_dimensions' : {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mozilla'},
@@ -2155,7 +2155,7 @@
   },
   'V8 Linux - arm - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access'},
@@ -2207,7 +2207,7 @@
   },
   'V8 Linux - arm - sim - debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access', 'shards': 3},
@@ -2260,7 +2260,7 @@
   },
   'V8 Linux - arm - sim - lite': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default', 'shards': 2},
@@ -2268,7 +2268,7 @@
   },
   'V8 Linux - arm - sim - lite - debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'v8testing', 'variant': 'default', 'shards': 4},
@@ -2276,7 +2276,7 @@
   },
   'V8 Linux - arm64 - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {'name': 'mjsunit_sp_frame_access'},
@@ -2289,7 +2289,7 @@
   },
   'V8 Linux - arm64 - sim - debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     # TODO(machenbach): Remove longer timeout when this builder scales better.
     'swarming_task_attrs': {
@@ -2305,7 +2305,7 @@
   },
   'V8 Linux - arm64 - sim - gc stress': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2322,7 +2322,7 @@
   },
   'V8 Linux - loong64 - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2335,7 +2335,7 @@
   },
   'V8 Linux - mips64el - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2349,7 +2349,7 @@
   },
   'V8 Linux - ppc64 - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2362,7 +2362,7 @@
   },
   'V8 Linux - riscv32 - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2375,7 +2375,7 @@
   },
   'V8 Linux - riscv64 - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2388,7 +2388,7 @@
   },
   'V8 Linux - s390x - sim': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2421,7 +2421,7 @@
   },
   'V8 Linux64 - arm64 - sim - no pointer compression': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 14400,
@@ -2436,7 +2436,7 @@
   # Clusterfuzz.
   'V8 NumFuzz': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 13800,
@@ -2458,7 +2458,7 @@
   },
   'V8 NumFuzz - TSAN': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 13800,
@@ -2518,7 +2518,7 @@
   },
   'V8 NumFuzz - debug': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'swarming_task_attrs': {
       'expiration': 13800,
@@ -2585,7 +2585,7 @@
   },
   'v8_numfuzz_rel': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -2602,7 +2602,7 @@
   },
   'v8_numfuzz_tsan_rel': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
@@ -2655,7 +2655,7 @@
   },
   'v8_numfuzz_dbg': {
     'swarming_dimensions': {
-      'os': 'Ubuntu-18.04',
+      'os': 'Ubuntu-22.04',
     },
     'tests': [
       {
-- 
2.35.1

